libs/core/pyproject.toml

```diff
@@ -44,8 +44,8 @@ python = ">=3.12.4"
 [tool.poetry.extras]
 
 [tool.ruff.lint]
-select = [ "ASYNC", "B", "C4", "COM", "DJ", "E", "EM", "EXE", "F", "FLY", "FURB", "I", "ICN", "INT", "LOG", "N", "NPY", "PD", "PIE", "Q", "RSE", "S", "SIM", "SLOT", "T10", "T201", "TID", "UP", "W", "YTT",]
-ignore = [ "COM812", "UP007", "S110", "S112",]
+select = [ "ASYNC", "B", "C4", "COM", "DJ", "E", "EM", "EXE", "F", "FLY", "FURB", "I", "ICN", "INT", "LOG", "N", "NPY", "PD", "PIE", "Q", "RSE", "RUF", "S", "SIM", "SLOT", "T10", "T201", "TID", "UP", "W", "YTT",]
+ignore = [ "COM812", "UP007", "RUF010", "RUF012", "S110", "S112",]
```

Comment:
RUF010 temporarily ignored because it woul create lots of conflicts for the PR with https://github.com/langchain-ai/langchain/pull/29334
RUF012 ignored because it doesn't play well with Pydantic